Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.19
+
+RUN go install github.com/registrobr/rdap-client@latest
+
+ENTRYPOINT ["/go/bin/rdap-client"]
```

Comment:
Some suggestions:
* Multi-stage build so the final image is smaller
* Add metadata to the generated image (allow injecting info)
* Disable DWARF generation and symbol table in the final binary to minimize the size

```dockerfile
#
# ===========
# Build image
# ===========
#
FROM golang:1.19
COPY . /go/src/github.com/registrobr/rdap-client
WORKDIR /go/src/github.com/registrobr/rdap-client
RUN mkdir /apps
RUN go build -mod=vendor -ldflags="-w -s" -o /apps/rdap-client

#
# ====================
# Final delivery image
# ====================
#
FROM runtime

COPY --from=builder /apps/* /apps/

ARG BUILD_DATE
ARG BUILD_VCS_REF
ARG BUILD_VERSION

ENV API_VERSION ${BUILD_VCS_REF}

LABEL org.label-schema.build-date=$BUILD_DATE \
      org.label-schema.description="RDAP client" \
      org.label-schema.name="rdap-client" \
      org.label-schema.schema-version="1.0" \
      org.label-schema.url="https://registro.br" \
      org.label-schema.vcs-url="https://github.com/registrobr/rdap-client" \
      org.label-schema.vcs-ref=$BUILD_VCS_REF \
      org.label-schema.vendor="NIC.br" \
      org.label-schema.version=$BUILD_VERSION

ENTRYPOINT ["/apps/rdap-client"]
```

Reply:
Hello @rafaeljusto !

Thank you so much for you review! I made something simples to attend a environment separation demand, but you suggestion is in fact far better.

Since i'm using podman to build - for good this time - image must be OCI compliant. I decided to use `FROM debian:stable-slim` since its small and very portable - with proper cleanups after installation of ca-certificates (Required by rdap-client) - and shares glibc compatibility. Also, I've added the missing `as builder` in first 'FROM' statement.

Besides choosing debian:stable-slim, I accepted you code entirely and